torchci/lib/bot/mergeBot.ts

```diff
@@ -102,7 +102,11 @@ function mergeBot(app: Probot): void {
         );
       }
     }
-    if (ctx.payload.comment.user.id == 54816060) {
+    const skipUsers = [
+      54816060, // pytorch-bot
+      97764156, // pytorchmergebot
+    ];
```

Comment:
Mind renaming it to `botAccounts` and move it to the top of the list? I.e. why we want this check only for rebase bot?

Reply:
it's not just for rebase bot; it's a little confusing because handleRebase is declared inline in the main handling function